### DISCUSSION TITLE
Download All Subreddit Media

### DISCUSSION THREAD

**Discussion by aminiqmal39** [AAndyProgram/SCrawler#139] [Q&A] [answered]
Hi, i'm still not understand how the program work. I just want to download all media from one subreddit and i don't even know where to put subreddit name in your program. Could you help me understand how to make it work?

**Comment by AAndyProgram** [comment, marked as answer]
https://github.com/AAndyProgram/SCrawler/wiki#add-user

![image](https://github.com/AAndyProgram/SCrawler/assets/88590076/412a8831-f0ca-4f9b-82ac-e44f95c29a31)

**Reply by aminiqmal39** [reply]
Why download data on user only download a few pic? not all.

**Reply by AAndyProgram** [reply]
Uncheck `Get user media only`, remove this channel, add and download it again

**Reply by aminiqmal39** [reply]
Lastly, it seem redgif not downloading. Maybe the bearer token not working or which bearer token should i use?
![image](https://github.com/AAndyProgram/SCrawler/assets/63519621/eba2ccb3-14c0-4299-a33e-a2a2c998f88a)

**Reply by AAndyProgram** [reply]
Bearer token is received and renewed automatically. Please post the profile URL you're trying to download.

**Reply by aminiqmal39** [reply]
Here the profile URL for testing. https://www.reddit.com/r/rape_hentai/

**Reply by AAndyProgram** [reply]
What should I test?

**Reply by aminiqmal39** [reply]
Redgifs not working.

**Reply by AAndyProgram** [reply]
Not confirmed. Redgifs downloader is fine. I don't see redgifs on this site, so I downloaded a redgifs profile. All videos are downloaded. Please, give me a profile and a specific post that includes redgifs that is not downloaded.
